tests/dummy/app/templates/denali-table.hbs

```diff
@@ -0,0 +1,20 @@
+<DenaliTable as |Table|>
+  <Table.tr as |Tr|>
+    <Tr.th @alignContent="right">Date</Tr.th>
```

Comment:
Weird to not capitalize `Tr.th`. Is it possible to yield the component directly instead of wrapping it in an obj? For some reason I feel like there's a problem with doing this but it would be nicer.